spec/zoneddatetime.html

```diff
@@ -589,6 +589,7 @@ <h1>Temporal.ZonedDateTime.prototype.with ( _temporalZonedDateTimeLike_ [ , _opt
           1. Perform ? Set(_fields_, *"offset"*, _offsetString_).
         1. Else,
           1. Set _offsetString_ to ? Get(_fields_, *"offset"*).
+          1. Set _offsetString_ to ? ToString(_offsetString_).
```

Comment:
```suggestion
          1. Set _offsetString_ to ? ToString(? Get(_fields_, *"offset"*)).
```

Reply:
sorry, I really does not like that. 
When we implement that in C++ inside chome, it will be much harder to follow....

Reply:
That's the most common pattern in the spec; creating a variable and then reassigning it is something that's typically avoided.

The spec is written far more like JS than like C++.